tests/cache/jobQueue/getHierarchy.js

```diff
@@ -0,0 +1,347 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/cache/jobQueue/getHierarchy.js
+ */
+'use strict';
+const jobQueue = require('../../../jobQueue/setup').jobQueue;
+const jobType = require('../../../jobQueue/setup').jobType;
+const expect = require('chai').expect;
+const supertest = require('supertest');
+const api = supertest(require('../../../index').app);
+const samstoinit = require('../../../cache/sampleStoreInit');
+const tu = require('../../testUtils');
+const rtu = require('../models/redisTestUtil');
+const constants = require('../../../api/v1/constants');
+const Subject = tu.db.Subject;
+const path = '/v1/subjects/{key}/hierarchy';
+const logger = require('../../../utils/activityLog').logger;
+
+describe(`api: GET using worker process ${path}`, () => {
+
+  before(() => {
+    tu.toggleOverride('enableWorkerProcess', true);
+    tu.toggleOverride('enqueueHierarchy', true);
+    require('../../../worker/jobProcessor');
```

Comment:
I assume you mean jobType.GET_HIERARCHY?
I wrote this before I merged the change that split out the job processors so that never occurred to me.
Doesn't seem like it should matter much either way, but I think it's easier to understand this way since the idea is to simulate a normal worker process.